### ISSUE TITLE
Error when Maven classpath includes non-*.jar entries

### ISSUE THREAD

**Issue by jglick** [johnpoth/jshell-maven-plugin#8]
If your classpath includes entries which are not named according to the expectations in https://github.com/openjdk/jdk/commit/9691449f5f17ed7d2e062a2ef6d7d4f4845000b7 then this mojo crashes. I discovered this while trying to use `-testClasspath` on a Jenkins plugin, which includes a WAR file for functional testing purposes—cannot load classes from it, but can be located in the `java.class.path` and then loaded in other ways using a `jetty:run` variant. Since `jshell` is strict about this, the mojo should automatically exclude any entries which would be rejected.

To reproduce:

```console
$ export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64 # on Ubuntu 19.10
$ git clone https://github.com/jenkinsci/workflow-api-plugin
…
$ cd workflow-api-plugin
$ mvn -ntp -DskipTests clean package
…
$ mvn com.github.johnpoth:jshell-maven-plugin:1.1:run
[INFO] Scanning for projects...
[INFO] 
[INFO] ------------< org.jenkins-ci.plugins.workflow:workflow-api >------------
[INFO] Building Pipeline: API 2.41-SNAPSHOT
[INFO] --------------------------------[ hpi ]---------------------------------
[INFO] 
[INFO] --- jshell-maven-plugin:1.1:run (default-cli) @ workflow-api ---
|  Welcome to JShell -- Version 11.0.6
|  For an introduction type: /help intro

jshell> org.jenkinsci.plugins.workflow.log.LogStorage.endStep()
$1 ==> "</span>"

jshell> [INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  49.822 s
[INFO] Finished at: 2020-03-26T10:45:03-04:00
[INFO] ------------------------------------------------------------------------
$ mvn com.github.johnpoth:jshell-maven-plugin:1.1:run -DtestClasspath
[INFO] Scanning for projects...
[INFO] 
[INFO] ------------< org.jenkins-ci.plugins.workflow:workflow-api >------------
[INFO] Building Pipeline: API 2.41-SNAPSHOT
[INFO] --------------------------------[ hpi ]---------------------------------
[INFO] 
[INFO] --- jshell-maven-plugin:1.1:run (default-cli) @ workflow-api ---
Invalid '--class-path' argument: /home/jglick/.m2/repository/org/jenkins-ci/main/jenkins-war/2.150.3/jenkins-war-2.150.3.war
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.280 s
[INFO] Finished at: 2020-03-26T10:45:27-04:00
[INFO] ------------------------------------------------------------------------
$ 
```

**Comment by johnpoth** [comment]
Thanks @jglick fixed with ec2ff909cc3190f515bdaa2228a79622441f4e46, thanks for the report !

**Comment by jglick** [comment]
Verified in 5b941dca7dc1e6f855f73c580668458713d846b0.
